src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.tsx

```diff
@@ -137,55 +109,23 @@ const List: FunctionComponent<ListProps> = ({
           <tr>
             <ThParent data-testid="parent" />
             <ThPriority data-testid="priority" />
-            <Th data-testid="sortId" onClick={onSort('id')}>
-              Id {renderChevron('id')}
-            </Th>
-            <Th data-testid="sortDaysOpen" onClick={onSort('days_open')}>
-              Dag {renderChevron('days_open')}
-            </Th>
-            <ThDate data-testid="sortCreatedAt" onClick={onSort('created_at')}>
-              Datum en tijd {renderChevron('created_at')}
-            </ThDate>
-            <ThSubcategory
-              data-testid="sortSubcategory"
-              onClick={onSort('sub_category,-created_at')}
-            >
-              Subcategorie {renderChevron('sub_category')}
+            <Th data-testid="sortId">Id</Th>
+            <Th data-testid="sortDaysOpen">Dag</Th>
+            <ThDate data-testid="sortCreatedAt">Datum en tijd</ThDate>
+            <ThSubcategory data-testid="sortSubcategory">
+              Subcategorie
```

Comment:
Is it possible to remove the data-testid attributes now that the sorting is done differently?

Reply:
I guess so. They don't seem to be used at all.